srcpkgs/golang-migrate/template

```diff
@@ -0,0 +1,23 @@
+# Template file for 'golang-migrate'
+pkgname=golang-migrate
+version=4.14.1
+revision=1
+wrksrc="migrate-${version}"
+build_style=go
+go_import_path="github.com/golang-migrate/migrate/v4/cmd/migrate"
+# https://github.com/golang-migrate/migrate/blob/af256da8ca1a8227cbe131eeafa6049b0154e4c4/Makefile#L1-L2
+go_build_tags="
```

Comment:
Do they have a makefile or something with these tags, so we don't have to repeat and remember to update them?

Reply:
Also this should be indented with spaces.

Reply:
They do have a Makefile.  If I want to build using their Makefile, will I have to forgo the `build_style=go`?  Or will it be fine to just override the parts I need (e.g. `do_build`) and it will be okay?

Re. indenting: I indented it with spaces at first, but then `xlint` complained at me.

Reply:
Do use the build style, just override the functions.

Multiline strings should definitely be indented with spaces (just one!), so let xlint complain if it will.

Reply:
Updated the template to override `do_build` and use their Makefile.  No longer have to worry about multiline strings since it's not needed anymore!